sigma/pipelines/panther/sdyaml_transformation.py

```diff
@@ -40,15 +36,24 @@ def apply(self, pipeline: ProcessingPipeline, rule: SigmaRule, query: Any) -> An
         if rule.level:
             res["Severity"] = rule.level.name
 
-        key = rule.logsource.product
-        log_type = LOG_TYPES_MAPPING.get(rule.logsource.product)
-        if log_type is None:
-            logging.error(f"Can't find LogTypes mapping for {key}")
+        log_types = self._detect_log_types(rule)
+        if len(log_types) == 0:
+            logging.error(f"Can't find ant LogTypes")
```

Comment:
```suggestion
            logging.error(f"Can't find any LogTypes")
```

Reply:
Also, do we want to return `None, False` here if we get into this state?

Reply:
if we add a `return` it would  return `- null` as result of rule conversion, so you can't see/use converted detection rules ( LogTypes is part of metadata and have no effect on detection)

Reply:
Makes sense -- thanks for the context!